solidity/test/BasicConsumer_test.js

```diff
@@ -160,17 +163,17 @@ contract('BasicConsumer', () => {
 
   describe('#withdrawLink', () => {
     beforeEach(async () => {
-      await link.transfer(cc.address, web3.toWei('1', 'ether'))
+      await link.transfer(cc.address, toWei('1', 'ether').toString())
       const balance = await link.balanceOf(cc.address);
-      assert.equal(balance.toString(), web3.toWei('1', 'ether'));
+      assert.equal(balance.toString(), toWei('1', 'ether').toString());
     })
 
     it('transfers LINK out of the contract', async () => {
       await cc.withdrawLink({from: consumer});
       const ccBalance = await link.balanceOf(cc.address);
-      const consumerBalance = await link.balanceOf(consumer);
+      const consumerBalance = bigNum(await link.balanceOf(consumer));
       assert.equal(ccBalance.toString(), '0');
-      assert.equal(consumerBalance.toString(), web3.toWei('1', 'ether'));
+      assert(consumerBalance.eq(toWei('1', 'ether')));
```

Comment:
This whole file seems to only have cosmetic changes. Is the motivation readability?

Reply:
@se3000 requested that `toWei` and `toUtf8` convenience functions be used in place of the `web3` calls, so we don't have to deal with this again if web3 decides to change their namespace layout/API again.